.ci/benchmarks.groovy

```diff
@@ -85,6 +93,14 @@ pipeline {
             }
           }
         }
+        success {
+          dir("${BASE_DIR}") {            
+            sendSlackReportSuccessMessage()
+          }
+        }
+        failure {
+          notifyBuildResult(slackComment: true, slackChannel: ${env.SLACK_CHANNEL})
```

Comment:
typo, use double quotes for the interpolation

```suggestion
          notifyBuildResult(slackComment: true, slackChannel: "${env.SLACK_CHANNEL})"
```

or 

```suggestion
          notifyBuildResult(slackComment: true, slackChannel: env.SLACK_CHANNEL)
```

Although, SLACK_CHANNEL is the default behaviour, see https://github.com/elastic/apm-pipeline-library/blob/0b1b3269e6a25bf50e5d78ed6102894c7f712fce/vars/notifyBuildResult.groovy#L66


```suggestion
          notifyBuildResult(slackComment: true)
```

The idea in this section is to send a slack message with the build failure only, correct?